### ISSUE TITLE
control-compiled.xsl fails to inherit XML 1.1

### ISSUE THREAD

**Issue by Arithmeticus** [xspec/xspec#989]
I have an xspec file written in XML 1.1 because it handles control characters, but the XSpec test fails because of those characters. It seems that the generated control-compiled.xsl file does not properly inherit the 1.1 value for the version pseudo-attribute. For background discussion [see this oXygen forum thread](https://www.oxygenxml.com/forum/post58464.html#p58464).

**Comment by AirQuick** [comment]
`saxon.custom.options=-xmlversion:1.1` ([Ant](https://github.com/xspec/xspec/search?q=saxon.custom.options&type=Wikis)) and `SAXON_CUSTOM_OPTIONS=-xmlversion:1.1` ([CLI](https://github.com/xspec/xspec/search?q=SAXON_CUSTOM_OPTIONS&type=Wikis)) are valid, but they deliberately take effect only at run time. I think enabling XML 1.1 at compile time needs to introduce a certain flag to `build.xml` and `bin/xspec.*` and/or `src/schemas/xspec.rnc`, and I'm afraid designing and implementing it will take some time.

For the meantime, you may possibly be able to work around with `codepoints-to-string()` perhaps together with [AVT](https://github.com/xspec/xspec/wiki/Writing-Scenarios#attributes-in-content) and [TVT](https://github.com/xspec/xspec/wiki/Writing-Scenarios#text-value-templates). For example:

```console
C:\xspec>type control.xspec
<?xml version="1.0" encoding="UTF-8"?>
<x:description xmlns:x="http://www.jenitennison.com/xslt/xspec"
    stylesheet="control.xsl">
    <x:scenario label="test">
        <x:call template="test"/>
        <x:expect label="test" expand-text="yes">'{codepoints-to-string(7)}'</x:expect>
    </x:scenario>
</x:description>
C:\xspec>set SAXON_CUSTOM_OPTIONS=-xmlversion:1.1

C:\xspec>bin\xspec.bat control.xspec
...
passed: 1 / pending: 0 / failed: 0 / total: 1
```

CCing @AlexJitianu just in case.

**Comment by AirQuick** [comment]
Implemented in #1141.
